src/NUnitFramework/framework/Interfaces/ICommandWrapper.cs

```diff
@@ -66,8 +66,8 @@ public interface IWrapSetUpTearDown : ICommandWrapper
 
     /// <summary>
     /// Objects implementing this interface are used to wrap
-    /// tests that can repeat. The implemeting object runs
-    /// each time before the test is repeats.
+    /// tests that can repeat. The implemeting command is run once,
+    /// invoking the chained commands as many times as defined.
```

Comment:
Sooo close! I have to point out the typo `implemeting` here. Also, "as defined" sounds to me a little more vague than saying "invoking the chained commands any number of times."